.obsidian/themes/Light & Bright/theme.css

```
@@ -0,0 +1,1094 @@
+}
+/* CALLOUTS */
+
+.callout {
+  overflow: hidden;
+  border-style: solid;
+  border-color: rgba(var(--callout-color), var(--callout-border-opacity));
+  margin:13px 0px;
+	padding:17px 27px 17px 37px;
+	border-radius:1em;
+	background-color:rgba(var(--callout-color), 5%);
+	border: transparent;
```

Biome (2.1.2)

[error] Unexpected shorthand property border after border-style



(lint/suspicious/noShorthandPropertyOverrides)